woocommerce/woocommerce-gutenberg-products-block
# Improve AI prompt for the header of the Hero Product 3 Split pattern

## What

Fixes #11832

## Why



This PR improves the prompt to generate a description that matches the image on the left. I'm not very expert with prompt engineering, so very happy to receive feedback :) 


### Prompt 1
```
I want to sell flowers.
```

![image](https://github.com/woocommerce/woocommerce-blocks/assets/4463174/11d612c4-6832-4e96-a082-d3fdea1790bf)

### Prompt 2
```
I want to sell candies
```

![image](https://github.com/woocommerce/woocommerce-blocks/assets/4463174/2e5030d5-2eaf-45a7-8be3-af9dda229324)

### Prompt 3 
```
Cabbage Acres is a 5th-generation family run farm outside of Portland, Oregon. We sell a variety of fresh produce and meats from our 1,000 acre organic farm. Our most popular products are:

Weekly CSA box of fresh fruit and vegetables
Free range, grass fed beef (ground beef, steaks, and more)
Free range, organic feed eggs from our happy chickens
Unlike most large-scale farms Cabbage Acres is a family operation as we’ve been farming the same land since the early 1900s. Our prices might be higher than the local grocery store but you’ll be sure to get high-quality, great-tasting produce right from your own community.
```

![image](https://github.com/woocommerce/woocommerce-blocks/assets/4463174/62debd00-9c69-40eb-b770-75f1d6c4ee4a)


## Testing Instructions

### Setup

Please test the feature on a WooExpress install instead of Jurassic Ninja, as the latter can be very limited in terms of server resources and unstable, as other installs can interfere with your own.

- If you already have a pre-configured install, remove the pre-existing WooCommerce Blocks Plugin and upload the one from this zip file: [woocommerce-gutenberg-products-block.zip](https://github.com/woocommerce/woocommerce-blocks/files/13437193/woocommerce-gutenberg-products-block.zip)
- Alternatively, create a new WooExpress install from scratch 
- Remove the pre-existing WooCommerce plugin via SSH and
- Install and activate WooCommerce from the following zip file: 
[woocommerce.zip](https://github.com/woocommerce/woocommerce-blocks/files/13415324/woocommerce.zip)
- Install and activate WooCommerce Blocks from the following zip file: [woocommerce-gutenberg-products-block.zip](https://github.com/woocommerce/woocommerce-blocks/files/13437193/woocommerce-gutenberg-products-block.zip)
- Install and activate WooCommerce Beta Tester (the plugin is available [within the monorepo](https://github.com/woocommerce/woocommerce/tree/trunk/plugins))
- Head over to `/wp-admin/tools.php?page=woocommerce-admin-test-helper` and enable `customize-store` feature flag:

<img width="1233" alt="Screenshot 2023-10-22 at 10 32 24" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/816867c6-34e5-40a1-a87d-f4a7cda46429">

### Test the CYS experience

0. Make sure you don't have any products in your store.
1. Head over to Home > Customize your Store:
 
<img width="1232" alt="Screenshot 2023-10-22 at 09 29 08" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/b5d34131-3edc-47ed-a8e4-8e2775ca1bbd">

2. Click on design with AI.
3. Type a description for your business.
4. Make sure that the header of the Hero Product 3 Split pattern matches with the image and the description of the business.


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|        |       |

## WooCommerce Visibility


Required:


## Checklist

Required:


Conditional:


## Changelog


> Add suggested changelog entry here.
